griptape/templates/engines/rag/modules/response/prompt/system.j2

```diff
@@ -0,0 +1,19 @@
+{% if before_system_prompt %}
+{{ before_system_prompt }}
+
+{% endif %}
+Always be truthful. Don't make up facts. You can answer questions by searching through text chunks. Use the following list of text chunks to respond. If there are no text chunks available or text chunks don't have relevant information respond with "I could not find an answer."
+
+{% if text_chunks and text_chunks|length > 0 %}
+{% for chunk in text_chunks %}
+Text chunk: """
+{{ chunk }}
+"""
```

Comment:
I know we do this in other engines too, but do the chunks belong in the system prompt? I always thought the system prompt was better for describing _how_ to do something ("answer questions by searching through text chunks"), and the user prompt for _what_ to do ("look through these specific chunks to answer this question").

Reply:
That's a good question. I think that's something we need to evaluate once we have evals. I suspect the result will depend on the underlying LLM. The nice thing about modular RAG is that we can easily try those different approaches by substituting response modules.